lk_inc.mk

```diff
@@ -1,10 +1,2 @@
 EMMC_BOOT := 1
 DISPLAY_2NDSTAGE := 1
-
-# it's not nice to rely on the original values here but it's the best we can do right now
```

Comment:
are you sure about that?
If your device uses non-standard addresses you may want to use the same